source/hassio/debugging.markdown

```diff
@@ -13,7 +13,7 @@ The following debug tips and tricks are for people who are running the Hass.io i
 
 ## {% linkable_title SSH access to the host %}
 
-Create an `authorized_keys` file in the root of your SD card with your public key. Once the device is booted, you can access your device as root over SSH on port 22222.
+Create an `authorized_keys` file in the root of your SD card with your public key. Once the device is booted, you can access your device as root over SSH on port 22222. The contents of the 'authorized_keys' file should be the contents of your 'id_rsa.pub' file.
```

Comment:
If you really don't want to reuse a file, than we should include in the docs the complete workflow to create a specific key for the hass.io installation.

As hass.io won't have any display output, I would also recommend moving this inside the "Installation" section. I didn't notice this at start, but I also just opened a pull request for something similar: https://github.com/home-assistant/home-assistant.github.io/pull/2870.